src/Microsoft.PowerShell.Security/security/AclCommands.cs

```diff
@@ -528,7 +525,7 @@ public static string[] GetAllCentralAccessPolicies(PSObject instance)
 
                 // Add CAP names and IDs to a string array.
                 string[] policies = new string[capCount];
-                NativeMethods.CENTRAL_ACCESS_POLICY cap = new NativeMethods.CENTRAL_ACCESS_POLICY();
+                NativeMethods.CENTRAL_ACCESS_POLICY cap;
```

Comment:
I think this can be removed here and instead be defined in the for loop body scope, where it is only used (similar to pCapId)/

Reply:
Actually, this code is inside of an `if !CORECLR` do you still want the change made?